artifacts/contracts/discrepancies/nonce/ChainedContracts.sol/ChainedContracts.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
These artifacts should not be added anymore after this commit https://github.com/hashgraph/hedera-smart-contracts/commit/dfaedf93e5dc857ce2451921c967230099e43ac5 (we only want contract-abi/). Please kindly rebase main and uncheck these artifacts from the git tree

Reply:
Done. Removed all json files that were uploaded

Reply:
So we don't want the `default` artifacts JSON files in the `artifacts/` folder to be checked in, but we still want to keep the `ABI-only artifact files` in the `contract-abi/` folder. Let me know if you have any question as I see this could be a little confusing.